apiserver/client/client.go

```diff
@@ -393,12 +392,13 @@ func (c *Client) ProvisioningScript(args params.ProvisioningScriptParams) (param
 		icfg.EnableOSRefreshUpdate = cfg.EnableOSRefreshUpdate()
 	}
 
-	result.Script, err = manual.ProvisioningScript(icfg)
+	result.Script, err = sshprovisioner.ProvisioningScript(icfg)
```

Comment:
I think we might want the ProvisioningScript function inside a linux-specific package, because it's independent of the method of execution. Hypothetically, you should be able to execute that script via os/exec just as well as via ssh.

What I imagined was: CLI calls into sshprovisioner, which detects the target machine's series, arch, etc. It then calls environs/manual.ProvisioningScript with that detected information, which calls into an OS-specific function to generate the script. i.e. something like:

   CLI -> sshprovisioner|winrmprovisioner -> environs/manual.ProvisioningScript -> environs/manual/{linux,windows,...}.ProvisioningScript

Leave it for now, but I think it may make sense to shuffle it around again later.